apix-impl/src/main/java/eu/xenit/apix/alfresco/metadata/NodeService.java

```diff
@@ -561,30 +563,22 @@ private QName toAlfrescoQName(String s) {
 
     }
 
+    //guessMimetype places a ContentStreamListener on ContentWriter and waits for the input to be written.
```

Comment:
Can you put the comment about `guessMimetype` above the `guessMimetype` call at L577? Same of L643.